WordPress/Classes/Services/SuggestionService.m

```diff
@@ -91,23 +82,84 @@ - (void)updateSuggestionsForSiteID:(NSNumber *)siteID
     }];
 }
 
+- (Autocompleter *)retrieveAutocompleterForSiteID:(NSNumber *)siteID
```

Comment:
We don't need to do it now but keep in mind for upcoming changes: given the amount of code we changed/added here we could have used this opportunity to migrate this code to Swift. :D

Reply:
Good point. Do we normally rewrite to Swift things that get significant changes?

Reply:
I decided to port `SuggestionService` and `Suggestion` to Obj-C: https://github.com/wordpress-mobile/WordPress-iOS/pull/14888

Reply:
@leandroalonso If you could take a look at that PR (https://github.com/wordpress-mobile/WordPress-iOS/pull/14888), that would be great! Once it's merged I'll update this PR, that should make this PR a bit smaller too.

Reply:
https://github.com/wordpress-mobile/WordPress-iOS/pull/14888 was merged so resolving this 👍